### PR TITLE
create shell resource operations objects

### PR DESCRIPTION
**Pull request by m-nash**

### CODE REVIEW

File: src/AutoRest.CSharp.V3/Output/Models/ClientBase.cs [hunk 1]
```diff
@@ -16,14 +16,12 @@ internal abstract class ClientBase: TypeProvider
...
         protected string ClientSuffix { get; }
 
-        private readonly BuildContext _context;
         private readonly TypeFactory _typeFactory;
 
         protected ClientBase(BuildContext context): base(context)
         {
             ClientSuffix = context.Configuration.AzureArm ? OperationsSuffixValue : ClientSuffixValue;
             _typeFactory = context.TypeFactory;
-            _context = context;
```
Comment (bquantump) [inline src/AutoRest.CSharp.V3/Output/Models/ClientBase.cs:26]: Was this removed because it is also in the base class?

Comment (m-nash) [reply src/AutoRest.CSharp.V3/Output/Models/ClientBase.cs:26]: yeah, made it protected in the baseclass

File: src/AutoRest.CSharp.V3/Properties/launchSettings.json [hunk 2]
```diff
@@ -1,236 +1,236 @@
...
-    },
-    "xml-service": {
-      "commandName": "Project",
-      "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestServerProjects\\xml-service"
-    },
-    "xms-error-responses": {
-      "commandName": "Project",
-      "commandLineArgs": "--standalone $(SolutionDir)\\test\\TestServerProjects\\xms-error-responses"
+    "profiles": {
+        "additionalProperties": {
```
Comment (bquantump) [inline src/AutoRest.CSharp.V3/Properties/launchSettings.json]: What are these changes, formatting?

**Comment by bquantump** [comment]
Is there any verification code or tests for this?

File: src/AutoRest.CSharp.V3/Properties/launchSettings.json (hunk 2 above)
Comment (m-nash) [reply src/AutoRest.CSharp.V3/Properties/launchSettings.json]: Fixed, was only supposed to add 1 configuration not modify the rest

**Pull request closed without merging**